windows/cross-compile/check-dlls

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+ifeq ($(ARCH),i686)
+WINE_DLL_DIR="/usr/lib/x86_64-linux-gnu/wine/i386-windows"
+endif
+ifeq ($(ARCH),x86_64)
+WINE_DLL_DIR="/usr/lib/x86_64-linux-gnu/wine/x86_64-windows" "/usr/lib/x86_64-linux-gnu/wine/i386-windows"
```

Comment:
Again, the i386 path is not correct, should be `/usr/lib/i386-linux-gnu/wine/i386-windows`.

Reply:
Done.
<!-- Reviewable comment -ODrYXUh8K3JvJvasBGN:b-896fix -->
<!-- Sent from Reviewable.io -->